src/watchful/client.py

```diff
@@ -1080,6 +1080,19 @@ def export_preview(mode: str = "ftc") -> Optional[Dict]:
 
     return api("export_preview", mode=mode)
 
+def export_project() -> http.client.HTTPResponse:
+    """
+    This function returns a consolidated version (a single *.hints file) of the
+    currently open project.
+    """
```

Comment:
Would this mean it is currently possible for a project to be made up of two or more hints files?

Reply:
No.  Hopefully this thread answered it:  https://watchfulio.slack.com/archives/C02NKPDNA0P/p1673276618423479?thread_ts=1673260763.951509&cid=C02NKPDNA0P

- Private hints files are one (1) *.hints file
- Shared, or "published" projects are a directory, with a `main` and `main.wip` file and a link-reference to the dataset(s).